hack/generator/pkg/astmodel/struct_definition.go

```diff
@@ -109,15 +82,15 @@ func (definition *StructDefinition) AsDeclarations() []ast.Decl {
 		},
 	}
 
-	if definition.description != "" {
+	if definition.description != nil {
 		declaration.Doc.List = append(declaration.Doc.List,
-			&ast.Comment{Text: "\n/* " + definition.description + " */"})
+			&ast.Comment{Text: "\n/*" + *definition.description + "*/"})
```

Comment:
Needs spaces around the content of the description

Reply:
At the moment this adversely affects CRD generation. Perhaps we can fix it in `controller-gen`.